packages/app/src/contacts/presenters/contact-attachment.presenter.test.ts

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { ContactAttachmentPresenter } from "App/contacts/presenters/contact-attachment.presenter"
+import { Contact } from "App/contacts/dto"
+
+const contact: Contact = {
+  id: "0",
+  firstName: "Sławomir",
+  lastName: "Borewicz",
+  primaryPhoneNumber: "+71 195 069 214",
+  secondaryPhoneNumber: "+1 911 020 12 23",
+  email: "example@mudita.com",
+  note: "sapiente rem dignissimos sunt",
+  ice: true,
+  favourite: false,
+  blocked: false,
+  firstAddressLine: "Malczewskiego 3, Warszawa",
+  secondAddressLine: "00-000 Somewhere",
+}
+
+describe("Method: toAttachment", () => {
+  test("Completed contact returns fully serialized attachment string", () => {
+    expect(ContactAttachmentPresenter.toAttachment(contact)).toEqual(
+      `Sławomir Borewicz
+
+
+[value] module.contacts.attachment.information
++71 195 069 214; +1 911 020 12 23; example@mudita.com
+[value] module.contacts.attachment.address
+Malczewskiego 3, Warszawa, 00-000 Somewhere
+[value] module.contacts.attachment.notes
+sapiente rem dignissimos sunt`
+    )
+  })
+
+  describe("Partly provided names", () => {
+    test("Contact without firstName returns attachment without first name", () => {
+      expect(
+        ContactAttachmentPresenter.toAttachment({
+          ...contact,
+          firstName: "",
+        })
+      ).toEqual(
+        `Borewicz
+
```

Comment:
Do we really need this two line empty space in the contact attachment? I think it will be better to remove them

Reply:
It was implemented according to design. But we can provide changes if you want

Reply:
Ok, so maybe we can do it as another PR after the discussion with the team